sqlite3_abi.nim

```diff
@@ -4,7 +4,21 @@
 import sqlite3_gen
 export sqlite3_gen
 
-proc sqlite3_bind_blob*(pstmt: ptr sqlite3_stmt, param: cint, value: pointer, n: cint, dispose: proc (v: pointer) {.cdecl.}): cint {.importc, cdecl.}
+proc sqlite3_bind_blob*(
+  pstmt: ptr sqlite3_stmt, 
+  param: cint, 
+  value: pointer, 
+  n: cint, 
+  dispose: proc (v: pointer) {.cdecl.}
```

Comment:
these disposers must actually be `{.cdecl, raises: [Defect].}`

Reply:
I did. Should I do the same with 3 procs called by sqlite3_create_function?

Also, does "raises: [Defect]" compatible both with Nim 1.2 and 1.6? Because we are now using 1.2 and plan to switch to 1.6